test/e2e/apps/daemon_set.go

```diff
@@ -381,7 +381,8 @@ var _ = SIGDescribe("Daemon set [Serial]", func() {
 	  rollback of updates to a DaemonSet.
 	*/
 	framework.ConformanceIt("should rollback without unnecessary restarts", func() {
-		if framework.TestContext.CloudConfig.NumNodes < 2 {
+		schedulableNodes := framework.GetReadySchedulableNodesOrDie(c)
+		if len(schedulableNodes.Items) < 2 {
 			framework.Logf("Conformance test suite needs a cluster with at least 2 nodes.")
```

Comment:
so here according to the k8s conformance we should just fail. 1 node clusters are not conformant.
to do that you can do:
```
schedulableNodes := framework.GetReadySchedulableNodesOrDie(c)
Expect(llen(schedulableNodes.Items)).To(BeNumerically(">", 1), "Conformance test suite needs a cluster with at least 2 nodes.")
```
^ untested.

Reply:
fixed differently,  for the test to run on a single node we need '>=1'